src/admin/config.yml

```diff
@@ -259,7 +259,7 @@ collections:
         default: []
         fields:
           - {label: "Title", name: "title", widget: "string", required: true}
-          - {label: "Content", name: "content", widget: "markdown", required: true, editor_components: ["image"]}
+          - {label: "Content", name: "content", widget: "markdown", required: true, editor_components: ["image", "file"]}
```

Comment:
The file component should be removed from the editor_components as custom widgets won't work in non-body fields.

Reply:
In vision-technology-service.md file, link to a media docx file is embedded in content field, and without specifying the file custom widget, I can't use + button to use the custom widget. Also, it seems like I can replace the existing link with the custom widget this way

Reply:
You are right. Custom widgets work well with markdown fields. If so, we could add this widget to other markdown fields. What do you think?

Reply:
I think we can add the file widget to only when it is necessary. Ideally, not specifying editor_component would be the best, because not specifying it would allow all custom widgets to be usable. We can also add additional editor components as we need in the future, in my opinion.

Reply:
Good point. I'm thinking to add "file" widget to fields that have `editor_components: ["image"]` defined.